### DISCUSSION TITLE
Tweak dual subtitles presentation?

### DISCUSSION THREAD

**Discussion by McMork** [smplayer-dev/smplayer#1028] [Q&A]
Hi, 
The fact that SMPlayer can display 2 sets of subtitles is great. 
I'd like to use it to help with language learning and recommend it to students.

But the default settings makes it pretty much useless, our eyes just don't work like that. It's impossible to catch the subtitles both at the top and bottom of the screen at the same time. 

I've tried moving the subtitles, but it only moves the primary subtitles and the secondary one just disappears.

Am I missing a setting? Is there any way to get SMPlayer to show the two sets of subtitles at the bottom of the screen?

I know I can combine 2 sets of subtitles into 1 using 3rd party tools, but that's too complicated to recommend to most people to consider. Also it seems that SMPlayer is so close to being the perfect solution, all I need to do is know how to move the subtitles.

Thanks!

**Comment by smplayer-dev** [comment]
You can move the secondary subtitles adding an option like this in preferences -> advanced -> mplayer/mpv -> options:
`--secondary-sub-pos=90`

(90 is the position, possible values 0-150)

**Comment by McMork** [comment]
Hi Ricardo!
First of all, thanks so much for developing SMPlayer, it's a great piece of software!!

Also, thanks very much for your advice. 

However, when I follow those instructions I get the error I've pasted below.
I'm running SMPlayer on Linux Mint 21.3 and the SMPlayer version is here:

> Version: 23.12.0 (revision 10207)
> Using Qt 5.15.3 (compiled with Qt 5.15.3)
> Using MPV 0.34.1



Error from log:
```
/usr/bin/mpv --no-quiet --terminal --no-msg-color --input-ipc-server=/tmp/smplayer-mpv-ab32 --msg-level=ffmpeg/demuxer=error --video-rotate=no --no-config --no-fs --hwdec=no --sub-auto=fuzzy --keepaspect --osd-level=1 --osd-scale=1 --osd-bar-align-y=0.6 --sub-ass --embeddedfonts --sub-ass-line-spacing=0 --sub-scale=1 --sub-font=Arial Black --sub-color=#ffffffff --sub-shadow-color=#ff000000 --sub-border-color=#ff000000 --sub-border-size=0.75 --sub-shadow-offset=2.5 --sub-font-size=50 --sub-bold=no --sub-italic=no --sub-margin-y=0 --sub-margin-x=20 --sub-codepage=ISO-8859-1 --sub-pos=100 --volume=100 --cache=auto --screenshot-template=cap_%F_%p_%02n --screenshot-format=jpg --screenshot-directory=/home/user/Downloads/Pictures/smplayer_screenshots --audio-pitch-correction=yes --volume-max=110 --secondary-sub-pos=90 --term-playing-msg=MPV_VERSION=${=mpv-version:}
INFO_VIDEO_WIDTH=${=width}
INFO_VIDEO_HEIGHT=${=height}
INFO_VIDEO_ASPECT=${=video-params/aspect}
INFO_VIDEO_FPS=${=container-fps:${=fps}}
INFO_VIDEO_FORMAT=${=video-format}
INFO_VIDEO_CODEC=${=video-format}
INFO_DEMUX_ROTATION=${=track-list/0/demux-rotation}
INFO_AUDIO_FORMAT=${=audio-codec-name}
INFO_AUDIO_CODEC=${=audio-codec-name}
INFO_AUDIO_RATE=${=audio-params/samplerate}
INFO_AUDIO_NCH=${=audio-params/channel-count}
INFO_LENGTH=${=duration:${=length}}
INFO_DEMUXER=${=current-demuxer:${=demuxer}}
INFO_SEEKABLE=${=seekable}
INFO_TITLES=${=disc-titles}
INFO_CHAPTERS=${=chapters}
INFO_TRACKS_COUNT=${=track-list/count}
METADATA_TITLE=${metadata/by-key/title:}
METADATA_ARTIST=${metadata/by-key/artist:}
METADATA_ALBUM=${metadata/by-key/album:}
METADATA_GENRE=${metadata/by-key/genre:}
METADATA_DATE=${metadata/by-key/date:}
METADATA_TRACK=${metadata/by-key/track:}
METADATA_COPYRIGHT=${metadata/by-key/copyright:}
INFO_MEDIA_TITLE=${=media-title:}
INFO_STREAM_PATH=${stream-path}
 --audio-client-name=SMPlayer --term-status-msg=STATUS: ${=time-pos} / ${=duration:${=length:0}} P: ${=pause} B: ${=paused-for-cache} I: ${=core-idle} VB: ${=video-bitrate:0} AB: ${=audio-bitrate:0} /home/user/Series/test.avi

Error parsing option secondary-sub-pos (option not found)
Setting commandline option --secondary-sub-pos=90 failed.
Exiting... (Fatal error)
```


Do you have any advice about how to resolve this?

**Comment by smplayer-dev** [comment]
That option was added recently. You need mpv 0.38.

**Reply by McMork** [reply]
Ok, thanks!! I'll try that once I figure out where to download 0.38.
